sigh/lib/sigh/runner.rb

```diff
@@ -216,8 +216,13 @@ def certificate_to_use
     # Downloads and stores the provisioning profile
     def download_profile(profile)
       UI.important "Downloading provisioning profile..."
-      profile_name ||= "#{profile_type.pretty_type}_#{Sigh.config[:app_identifier]}.mobileprovision" # default name
-      profile_name += '.mobileprovision' unless profile_name.include? 'mobileprovision'
+      profile_name ||= "#{profile_type.pretty_type}_#{Sigh.config[:app_identifier]}"
+
+      if Sigh.config[:platform].to_s == 'tvos'
+        profile_name += "_tvos"
+      end
+
+      profile_name += '.mobileprovision'
```

Comment:
ohhh .... i request to remove this, as i thought its not needed as it is always true - @KrauseFx  you mean there is a case where profile_name could already contain `.mobileprovision` - i didn't see any possibilities for that. (in sigh/runner)